### PR TITLE
Embed API: Glossary terms sharing description (Sphinx)

### PR DESCRIPTION
**Pull request by humitos**
I wasn't able to update https://github.com/readthedocs/readthedocs.org/pull/9984 by pushing my changes, so I went ahead and created a new PR for it that includes my changes as well 🤷🏼 cc @jpfeuffer

Closes https://github.com/readthedocs/sphinx-hoverxref/issues/97

### CODE REVIEW

File: readthedocs/embed/v3/views.py
Comment (benjaoming) [inline readthedocs/embed/v3/views.py:228]: why not `break`?

Comment (humitos) [reply readthedocs/embed/v3/views.py:228]: Just because I'm dumb 😝 . I fixed this. Thanks!

**Comment by jpfeuffer** [comment]
Awesome. This will make our docs much nicer! Thank you. I don't care about the closed PR of course.

**Comment by humitos** [comment]
I fixed the tests. This is ready to merge once I get an approve.

File: readthedocs/embed/v3/views.py
Comment (ericholscher) [inline readthedocs/embed/v3/views.py:228]: Isn't the `iteration >= 5` already provided by the `while iteration < 5`?

File: readthedocs/embed/v3/views.py
Comment (ericholscher) [inline readthedocs/embed/v3/views.py:223]: Should we raise an error here? What is "not produce correct results"?

**Review by ericholscher — APPROVED** [review]
Seems fine?

File: readthedocs/embed/v3/views.py
Comment (benjaoming) [reply readthedocs/embed/v3/views.py:228]: indeed, the first part of the condition can be removed, but it's also harmless.

File: readthedocs/embed/v3/views.py
Comment (humitos) [reply readthedocs/embed/v3/views.py:223]: I haven't tested this case. However, I think it's not possible to create a glossary entry without a description, tho

**Pull request merged**
